packages/dev/core/src/Engines/Extensions/engine.cubeTexture.ts

```diff
@@ -365,6 +365,9 @@ ThinEngine.prototype.createCubeTextureBase = function (
     const extension = forcedExtension ? forcedExtension : lastDot > -1 ? rootUrlWithoutUriParams.substring(lastDot).toLowerCase() : "";
 
     let loader: Nullable<IInternalTextureLoader> = null;
+    if (ThinEngine._TextureLoaders.length === 0) {

```

Comment:
This is valid if you specify a list of textures as it won t use external loaders.